xilem_web/src/context.rs

```diff
@@ -29,6 +29,13 @@ pub struct ViewCtx {
     id_path: Vec<ViewId>,
     app_ref: Option<Box<dyn AppRunner>>,
     pub(crate) fragment: Rc<web_sys::DocumentFragment>,
+    #[cfg(feature = "hydration")]
+    hydration_node_stack: Vec<web_sys::Node>,
+    #[cfg(feature = "hydration")]
+    is_hydrating: bool,
+    #[cfg(feature = "hydration")]
+    pub(crate) templates:
+        crate::vecmap::VecMap<std::any::TypeId, (web_sys::Node, Rc<dyn std::any::Any>)>,
```

Comment:
Maybe
```suggestion
        VecMap<TypeId, (web_sys::Node, Rc<dyn Any>)>,
```
and at the top

```rust
#[cfg(feature = "hydration")]
use crate::vecmap::VecMap;
#[cfg(feature = "hydration")]
std::any::{Any, TypeId};
```